nslsii/devices.py

```diff
@@ -49,7 +49,12 @@ def set(self, val):
         enums = self.status.enum_strs
 
         def shutter_cb(value, timestamp, **kwargs):
-            value = enums[int(value)]
+            try:
+                value = enums[int(value)]
+            except (ValueError, TypeError):
```

Comment:
Why so permissive? Should we check that the value is one of the expected enum types?

Reply:
Because I want to avoid having more than one cycle of getting a weighing version to the floor, we can make it stricter later.